infra/terraform/platform-base/main.tf

```diff
@@ -29,12 +29,25 @@ module "cluster_dns" {
   root_zone_id     = "${data.terraform_remote_state.global.platform_dns_zone_id}"
 }
 
+module "auth0" {
+  source = "../modules/auth0"
+
+  auth0_api_client_id          = "${var.auth0_api_client_id}"
+  auth0_api_client_secret      = "${var.auth0_api_client_secret}"
+  auth0_tenant_domain          = "${var.oidc_provider_domain}"
+  aws_account_id               = "${var.aws_account_id}"
+  env                          = "${terraform.workspace}"
```

Comment:
Minor thing, but I always put `env` at the top of the variable list.

Reply:
why?

Reply:
Just consistency - pretty much every module has that passed in, so I put it at the top so you can easily see that there's a common identifier used by each module.